integration-tests/test-suite/partials/layout.xml

```diff
@@ -24,6 +24,7 @@
 	<test name="S-comparator-Layout">
```

Comment:
I'm missing here some tests, that will check page with auto-height detection and fixed-height, e.g.:
- there is always changing element at the bottom of a page, scenario:
  - `F` with auto-height detection
  - `S` with fixed-height (let's say 100px), because screenshot does not contain this change

Reply:
Added - https://github.com/Cognifide/aet/pull/294/commits/0ae1f701dff30af3c95fe8debe3d2c3ef3c3d9a4